testing/upload_validation.ts

```diff
@@ -343,6 +344,24 @@ const defaultAuthenticationValidators: Record<AuthenticationType, z.ZodTypeAny>
     }).optional(),
     ...baseAuthenticationValidators,
   }),
+  [AuthenticationType.Custom]: zodCompleteStrictObject<CustomAuthentication>({
+    type: zodDiscriminant(AuthenticationType.Custom),
+    params: z
+      .array(
+        zodCompleteStrictObject<CustomAuthentication['params'][number]>({
+          name: z.string(),
+          description: z.string().optional(),
```

Comment:
The TypeScript for this field isn't optional. Should make these match, whatever you preference is.

Reply:
oops good catch, will make it required because it's the only indication in the UI for user auth for what the user should enter here lol